services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/oauth/OpenIDConnectManagerImpl.java

```diff
@@ -384,18 +384,26 @@ public UserAuthorization getUserAuthorization(String oauthToken) {
 
 		// userId is used to retrieve the user info
 		String userId = getUserIdFromPPID(ppid, oauthClientId);
-
-		UserAuthorization result = new UserAuthorization();
-		UserInfo userInfo = userManager.getUserInfo(Long.parseLong(userId));
+		// If the user belongs to the admin group they are an admin
+		Set<Long> groups = userManager.getUserGroups(Long.parseLong(userId));
+		
+		// Check to see if the user is an Admin
+		boolean isAdmin = groups.contains(AuthorizationConstants.BOOTSTRAP_PRINCIPAL.ADMINISTRATORS_GROUP.getPrincipalId());
+		// we don't let clients besides Synapse itself have admin access
+		boolean adminAccessAllowed = oauthClientId.equals(AuthorizationConstants.SYNAPSE_OAUTH_CLIENT_ID);
```

Comment:
Double check that the constant is a string

Reply:
Done.